docs/src/js/ds6/main.js

```diff
@@ -118,3 +118,41 @@ nodeListToArray(document.querySelectorAll('.listbox')).forEach(function(el, i) {
         }
     });
 });
+
+var mobileMediaQuery = 'all and (max-width: 600px)';
```

Comment:
Some comments might be good to describe what's going on in these blocks.

Reply:
I'll add some comments.